docs/allow-operator-to-group.md

```diff
@@ -0,0 +1,44 @@
+```ts
+import { ApiPromise, WsProvider, Keyring } from "@polkadot/api";
+import { blake2AsHex } from "@polkadot/util-crypto";
+
+const ONE_AVT = BigInt("1000000000000000000")
+
+async function main(): Promise<void> {
+  const wsProvider = new WsProvider("ws://localhost:9947"); wss://ewx-dev-parachain-aule-qb9wx41jvm.energyweb.org/ws
```

Comment:
Should this part be in a comment?
`wss://ewx-dev-parachain-aule-qb9wx41jvm.energyweb.org/ws`